cmd/agent/app/flare.go

```diff
@@ -81,12 +90,24 @@ func makeFlare(caseID string) error {
 		logFile = common.DefaultLogFile
 	}
 
-	var filePath string
+	var profile *flare.Profile
 	var err error
+	if profiling >= 30 {
+		fmt.Fprintln(color.Output, color.BlueString("Creating a %ds performance profile.", profiling))
+		profile, err = createPerformanceProfile()
+		if err != nil {
+			fmt.Fprintln(color.Output, color.RedString(fmt.Sprintf("Could not collect performance profile: %s", err)))
+			return err
+		}
+	} else {
+		fmt.Fprintln(color.Output, color.YellowString("Profiling not configured or below 30 seconds, skipping it."))
```

Comment:
The problem with exiting with error is that by default it is set as an invalid time range (0), because profiling is disabled by default. Should I change the default for this?

Reply:
Nvm I'll just make the default -1 lol